src/emulator/downloadableEmulators.ts

```diff
@@ -360,15 +366,19 @@ async function _runBinary(
     const logger = EmulatorLogger.forEmulator(emulator.name);
     emulator.stdout = fs.createWriteStream(getLogFileName(emulator.name));
     try {
-      emulator.instance = childProcess.spawn(command.binary, command.args, {
+      const opts: childProcess.SpawnOptions = {
         env: { ...process.env, ...extraEnv },
         // `detached` must be true as else a SIGINT (Ctrl-c) will stop the child process before we can handle a
         // graceful shutdown and call `downloadableEmulators.stop(...)` ourselves.
         // Note that it seems to be a problem with gRPC processes for which a fix may be found on the Java side
         // related to this issue: https://github.com/grpc/grpc-java/pull/6512
         detached: true,
         stdio: ["inherit", "pipe", "pipe"],
-      });
+      };
+      if (command.shell && utils.IS_WINDOWS) {
+        opts.shell = true;
+      }
+      emulator.instance = childProcess.spawn(command.binary, command.args);
```

Comment:
I think we should pass `opts` here(?). Something like 
```ts
emulator.instance = childProcess.spawn(command.binary, command.args, opts);
```

Reply:
Derp, good catch.